packages/components/pie-notification/src/index.ts

```diff
@@ -42,8 +42,13 @@ export * from './defs';
  * @event {CustomEvent} pie-notification-supporting-action-click - When the notification supporting action is clicked.
  * @event {CustomEvent} pie-notification-close - When the notification is closed.
  * @event {CustomEvent} pie-notification-open - When the notification is opened.
+ * @slot - Default slot
+ * @slot icon - The icon slot
  */
 export class PieNotification extends LitElement implements NotificationProps {
+    @state()
+        hasExternalIcon = false;
```

Comment:
Is this for internal use only? If so, we should probably mark is as `private` (and fix the indentation 😄)